### PR TITLE
Fix Raspberry Pi Docker Builds (#4012)

### LINKED ISSUE #4012: Fix Raspberry Pi Docker Builds
The purpose of this issue is to fix the docker ARM builds to work with Raspberry Pi. Currently they do not work for it which is what I'm planning to fix.

Things Needed To Change To Get It Working:
- [x] Merge in #3892 to help with speed as that is causing a lot of issues
- [x] Change the `phpmyadmin/phpmyadmin` image to just `phpmyadmin` for ARM Docker File
- [x] Disable CouchDB on 32-bit ARM builds as it isn't supported (we might want to create 2 different docker files, one for 32 bit and one for 64 bit)
- [x] Convert from regular node SASS to dart SASS as regular node SASS isn't supported on ARM
- [ ] Fix and/or disable CCDA service

### PR DESCRIPTION
**Pull request by tywrenn**
Fixes #4012

#### Short description of what this resolves:
Fixes ARM build issues (using Raspberry Pi 4 with Raspbian 32 bit).

### CODE REVIEW

**Comment by tywrenn** [comment]
@bradymiller The CCDA service is breaking ARM for me. Is there a way to disable it temporarily until we can make it compatible or...?

**Comment by bradymiller** [comment]
hi @tywrenn , Is that for 32bit or 64bit or both. btw, the ldap dockers now support arm(both v7 and 64); just built those couple hours ago to dockerhub.

File: docker/development-easy-arm/docker-compose.yml [hunk 1]
```diff
@@ -1,107 +1,85 @@
...
+    - nodemodules:/var/www/localhost/htdocs/openemr/node_modules:rw
+    - vendordir:/var/www/localhost/htdocs/openemr/vendor:rw
+    - ccdamodule:/var/www/localhost/htdocs/openemr/ccdaservice:rw
+    - logvolume:/var/log
+    environment:
+      DEBUG_COLORS: "true"
+      TERM: xterm-256color
+      COLORTERM: truecolor
+      MYSQL_HOST: mysql
+      MYSQL_ROOT_PASSWORD: root
```
Comment (bradymiller) [inline docker/development-easy-arm/docker-compose.yml:36]: why was this line modified?

File: docker/development-easy-arm/docker-compose.yml [hunk 2]
```diff
@@ -1,107 +1,85 @@
...
+      MYSQL_USER: openemr
+      MYSQL_PASSWORD: openemr
```
Comment (bradymiller) [inline docker/development-easy-arm/docker-compose.yml:38]: why was this line modified?

File: docker/development-easy-arm/docker-compose.yml [hunk 3]
```diff
@@ -1,107 +1,85 @@
...
+    depends_on:
+    - mysql
+  phpmyadmin:
+    restart: always
+    image: phpmyadmin
+    ports:
+    - 8310:80
+    environment:
+      PMA_HOSTS: mysql
+  #couchdb is disabled on this file due to incompatibility with 32 bit RPI
```
Comment (bradymiller) [inline docker/development-easy-arm/docker-compose.yml:68]: lets put comments like this at the top of the file

File: docker/development-easy-arm/docker-compose.yml (hunk 1 above)
Comment (tywrenn) [reply docker/development-easy-arm/docker-compose.yml:36]: The mariadb version that this uses has that variable set to PASSWORD instead of PASS. This might need to be forwarded to the docker differently but I'm not sure.

File: docker/development-easy-arm/docker-compose.yml (hunk 2 above)
Comment (tywrenn) [reply docker/development-easy-arm/docker-compose.yml:38]: The mariadb version that this uses has that variable set to PASSWORD instead of PASS. This might need to be forwarded to the docker differently but I'm not sure.

**Comment by tywrenn** [comment]
@bradymiller yeah ccda was hitting me with libxmljs issues but if that was solved then I could probably update ccda and fix it

File: docker/development-easy-arm/docker-compose.yml (hunk 1 above)
Comment (bradymiller) [reply docker/development-easy-arm/docker-compose.yml:36]: These are only environment settings for the OpenEMR docker, though. Would change them back or else the autoinstall will break.

File: docker/development-easy-arm/docker-compose.yml (hunk 2 above)
Comment (bradymiller) [reply docker/development-easy-arm/docker-compose.yml:38]: see above

File: docker/development-easy-arm/docker-compose.yml (hunk 1 above)
Comment (tywrenn) [reply docker/development-easy-arm/docker-compose.yml:36]: How does that get set from the openemr volume to the mariadb volume then? I'm no expert but if it's different in the openenr volume script it would definitely fail to run.

Comment (bradymiller) [reply docker/development-easy-arm/docker-compose.yml:36]: Environment variables only work within the docker they are set in. The mariadb docker is it's own docker with one environment setting (to see the root password). When the OpenEMR docker installs, it uses the `MYSQL_ROOT_PASS` environment setting as the credential to be able to connect to mariadb and then do its thing.
Shared volumes are a separate thing and I wouldn't worry about those (they can drive one crazy if try to figure out what all the shared volumes are doing in the easy dev environment :) )

File: package.json [hunk 4]
```diff
@@ -38,11 +38,11 @@
     "gulp": "^4.0.2",
     "gulp-append-prepend": "^1.0.8",
     "gulp-csso": "^4.0.1",
+    "gulp-dart-sass": "^1.0.2",
     "gulp-if": "^3.0.0",
     "gulp-inject-string": "^1.1.2",
     "gulp-postcss": "^9.0.0",
     "gulp-rename": "^2.0.0",
-    "gulp-sass": "^4.1.0",
```
Comment (bradymiller) [inline package.json:45]: @tywrenn , Is the transfer from gulp-sass to gulp-dart-sass testing well. If so, I may separate this out as a quick separate PR to get into the codebase so can then do a buildx with the arm64 for the openemr:6.0.0 docker.

**Comment by tywrenn** [comment]
Closed due to my changes already being added
